cmake/dependencies.cmake

```diff
@@ -93,6 +93,8 @@ if(THREADING STREQUAL "OMP")
 endif()
 
 ## TBB package
+# Fix downloading TBB package while using OpenCV package from third party server
+set(THIRDPARTY_SERVER_PATH "https://download.01.org/opencv/master/openvinotoolkit")
```

Comment:
There is no 4.5.5 version on download.01.org server

Reply:
@pramotowski

Reply:
4.5.5 ocv package is located at https://releases.ti.intel.com/Releases/opencv/thirdparty/linux/opencv/opencv_4.5.5-131.txz, that's why you have to set IE_PATH_TO_DEPS env pointing on it.

Although openvino building requires also TBB package, that is located here - https://download.01.org/opencv/master/openvinotoolkit/thirdparty/linux/tbbbind_2_5_static_lin_v2.tgz
The fix allows downloading both from releases.ti and download.01

Reply:
set(THIRDPARTY_SERVER_PATH "https://download.01.org/opencv/master/openvinotoolkit") - > i think you left the old path which does not contain the file

Reply:
> set(THIRDPARTY_SERVER_PATH "https://download.01.org/opencv/master/openvinotoolkit") - > i think you left the old path which does not contain the file

Without THIRDPARTY_SERVER_PATH set as "https://download.01.org/opencv/master/openvinotoolkit" linux job is unable to download TBB package, that result as fail.
I'm not sure that I understand "not contain the file" part, can you explain that?

Reply:
@alexbiris   @dstefan81
This is P1 stopper for our team. Its open for too long already.
We need aligned solution. Piotr proposed this basing on experiments from your earlier feedback.
Please work with Piotr to provide better/more proper solution if this one is missing anything.

Current workaround with using fork from fork from fork is not acceptable.
Help us to solve the problem.